code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -954,6 +954,11 @@
 	gib()
 	return
 
+/mob/living/silicon/robot/update_canmove() // to fix lockdown issues w/ chairs
+    if(lockcharge)
+        canmove = 0

```

Comment:
This looks suspicious. Has this been tested?

I fear it may never be set back to 1 this way?